frontend/app/model/RichEvent.scala

```diff
@@ -109,7 +116,12 @@ object RichEvent {
 
     val tags = Nil
 
-    val metadata = guLiveMetadata
+    val metadata = {
+      val highlight = contentOpt.map(c => HighlightsMetadata("Read more about this event", c.webUrl))
+        .orElse(Some(HighlightsMetadata("Watch highlights of past events", Config.guardianMembershipUrl + "#video")))
```

Comment:
maybe extract `HighlightsMetadata("Watch highlights of past events", Config.guardianMembershipUrl + "#video"))` in a val called `fallbackHighlights` or some such?